subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1744,6 +1743,16 @@ void ull_conn_done(struct node_rx_event_done *done)
 	}
 }
 
+void ull_conn_lll_tx_demux_sched(struct lll_conn *lll)
+{
+	static memq_link_t link;
+	static struct mayfly mfy = {0U, 0U, &link, NULL, tx_demux};
+
+	mfy.param = HDR_LLL2ULL(lll);
+
+	mayfly_enqueue(TICKER_USER_ID_LLL, TICKER_USER_ID_ULL_HIGH, 1U, &mfy);
+}
+
```

Comment:
Why isn't this code guarded by `defined(CONFIG_BT_CTLR_LOW_LAT_ULL)` similar to `tx_demux_sched` is guarded by `!defined(CONFIG_BT_CTLR_LOW_LAT_ULL)`